src/Enum.php

```diff
@@ -127,8 +129,16 @@ private static function resolveDefinition(): array
 
         $valueMap = static::values();
 
+        if (self::arrayHasDuplicates($valueMap)) {
```

Comment:
This doesn't prevent this case:

```php
/** 
 * @method static self value_1()
 * @method static self value_2()
 */

$valueMap = [
  'value_1' => 'value_2',
]
```

Which will result in a final enum definition array of two times the `value_2` value.

Reply:
Do we really need to prevent that though? I can't imagine anyone every doing that.

Reply:
My opinion: do it right/full or nothing.^^
So we can drop this check in it's whole if we only prevent one single source of failure. Having this exception and check but not really protecting from duplicates gives a false safety for me.

I think that this check would be easier with a cache array instead of the `EnumDefinition` class - or let the `EnumDefinition` implement `ArrayAccess` interface to still be able to use `array_column()` for such checks.

Reply:
https://github.com/spatie/enum/pull/59/commits/d90cd1199c5658d23e8f898f510e6303e3a9735d